WordPress/src/main/res/values/strings.xml

```diff
@@ -2962,5 +2962,6 @@ translators: sample content for "Services" page template -->
     <!-- translators: sample content for "About" page template -->
     <string name="gutenberg_native_what_people_say" tools:ignore="UnusedResources">What People Say</string>
     <string name="gutenberg_native_wordpress_media_library" tools:ignore="UnusedResources">WordPress Media Library</string>
+    <string name="content_description_logo">WordPress Logo</string>
```

Comment:
@renanferrari It looks like this string is added in the auto-generated section. As a result, it got removed during the code freeze [here](https://github.com/wordpress-mobile/WordPress-Android/commit/b4c9c0de94c472af795b5c877e512c8e24c06ebf#diff-96c14faed2c61eacb909ae0349b44cefL3097). We need to add new strings [above this line](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/strings.xml#L2856) otherwise it'll get removed by [this script](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/tools/merge_strings_xml.py).

Technically, you can add it to the very end, but I don't think we should do that and keep all application strings together.

Reply:
Thanks for letting me know of the issue and taking care of it, @oguzkocer! Really appreciate it.

I believe this string was added via Android Studio GUI and automatically placed at that position in the `strings.xml` file by the IDE. I thought it was curious that it would place it above the `<!--/Autogenerated:Gutenberg Native-->` line instead of below it (at the very end of the file) but I just tested this with Android Studio 4.0.1 and that's the behavior I got. This is usually not relevant as I'd normally go back to the `strings.xml` file and move the newly created string to a more appropriate place but I must have forgot to do so this time around.

I'm definitely not aware of the whole context behind this auto-generated section, but if possible, I'd suggest moving it into its own file, to prevent more issues like these in the future.

Reply:
As far as I am aware that's not possible at this time. On the contrary, these are the Gutenberg strings that has to be merged to the main `strings.xml` file for Glotpress to work. Let's cc @loremattei in case I am missing something.